.github/workflows/generateChangelog.yml

```diff
@@ -0,0 +1,32 @@
+name: Generate Changelog
+
+on:
+  workflow_call:
+
+jobs:
+  generate_changelog:
+    name: 'Generate Changelog'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: 'develop'
+          ssh-strict: false
+          token: ${{ secrets.IDEE_GH_TOKEN }}
+          fetch-depth: '0'
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+      - uses: ./.github/actions/gitConfig
+        with:
+          email: ${{ secrets.IDEE_GH_EMAIL }}
+      - name: Set NPM at the correct version for Lerna
+        run: npm install -g npm@9.8.1
```

Comment:
was this required or did you carry this over from somewhere?

Reply:
I copied them from createReleaseBranch.yml. both npm and lerna are used in s`alesforcedx-vscode/scripts/create-release-branch.js`

Reply:
b/c we're running it as an npm script node has access to the project dependancies (including lerna and shell js). I think you can remove this and line 28.  Give it a try on your fork and if it fails weirdly I'll give up and just 👍